README.md

```diff
@@ -57,6 +57,26 @@ pnpm run build:preview
 npx http-server ./dist
 ```
 
+Note that changes to config or build scripts will require re-running `pnpm start`.
```

Comment:
**suggestion**: I would link out to the build and config folders.

I also think that this could use some added emphasis like

## 🗒️ Note:

because it's something people easily forget. But again just a suggestion.

Reply:
I like it!

but how would we say build = lib folder (easy part) -- maybe we just rename this to build and then link.
and then for config, it's.. every other js file?

Maybe I can have a go at making the organization of this repo making more sense in a followup PR :thinking:

Reply:
Or maybe like `config files like <example config file>`

Reply:
https://github.com/CrowdStrike/tailwind-toucan-base/pull/89